subsys/net/ip/net_shell.c

```diff
@@ -3307,85 +3303,6 @@ static int cmd_net_route(const struct shell *shell, size_t argc, char *argv[])
 	return 0;
 }
 
-#if defined(CONFIG_INIT_STACKS)
-extern K_THREAD_STACK_DEFINE(_interrupt_stack, CONFIG_ISR_STACK_SIZE);
-extern K_THREAD_STACK_DEFINE(sys_work_q_stack,
-			     CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE);
-#endif
-
-static int cmd_net_stacks(const struct shell *shell, size_t argc,
```

Comment:
Instead of fully removing this shell command, we could leave if for couple of releases in order to give the user a hint to enable kernel shell to get the same info. So I was thinking something like this:
```
static int cmd_net_stacks(const struct shell *shell, size_t argc,
			  char *argv[])
{
#if !defined(CONFIG_KERNEL_SHELL)
	PR("Enable CONFIG_KERNEL_SHELL and type \"kernel stacks\" to see stack information.\n");
#else 
	PR("Type \"kernel stacks\" to see stack information.\n");
#endif 
}
```

Reply:
sounds good, done.